server/server.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/allegro/bigcache"
+)
+
+const (
+	// base HTTP paths.
+	apiVersion  = "v1"
+	apiBasePath = "/api/" + apiVersion + "/"
+
+	// path to cache.
+	cachePath = apiBasePath + "cache/"
+)
+
+var (
+	lifeWindow int64
+	port       int
+	logfile    string
+
+	// cache-specific settings.
+	cache  *bigcache.BigCache
+	config = bigcache.Config{
+		LifeWindow: time.Duration(lifeWindow) * time.Minute,
```

Comment:
Why it's not configurable? https://golang.org/pkg/flag/#DurationVar

Reply:
I didn't realise there was a time.Duration option in that package, to be honest. I will update with that.